README.md

```diff
@@ -23,12 +23,19 @@ started:
 
 **PyTorch/XLA is now on PyPI!**
 
-To install PyTorch/XLA a new TPU VM:
+To install PyTorch/XLA stable build in a new TPU VM:
 
 ```
 pip install torch~=2.3.0 torch_xla[tpu]~=2.3.0 -f https://storage.googleapis.com/libtpu-releases/index.html
 ```
 
+To install PyTorch/XLA nightly build in a new TPU VM:
+
+```
+pip3 install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
+pip3 install https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-nightly-cp310-cp310-linux_x86_64.whl
```

Comment:
does this usually works for you? I usually use our build of xla whl to be safe. If it usually works for you then maybe we should do this instead.

Reply:
On my v4-8, I used the above command and ran a resnet model. It seems running fine: https://gist.github.com/vanbasten23/9ae299668e8eba8d71d04c350355ac94